docs/development.md

```diff
@@ -129,7 +129,7 @@ can still be done if needed.
 2. Run bors locally, and configure environment variables and/or command-line parameters for it:
    - Set `APP_ID` to the ID of the created GitHub app.
    - Set `WEBHOOK_SECRET` to the webhook secret of the app.
-   - Set `PRIVATE_KEY` to the private key of the app.
+   - Set `PRIVATE_KEY` to the private key of the app. **Note:** If you face an error related to the private key, you need to add `\n` at the beginning of environment variable: `\n-----BEGIN RSA PRIVATE KEY`
```

Comment:
I tried it in my terminal and it was enough to `export PRIVATE_KEY=<paste from GitHub>`. But the important part was then using `"${PRIVATE_KEY}"` rather than just `${PRIVATE_KEY}` if I explicitly referred to the env. var. later.

This note seems a bit random, to be honest 😅 Did you figure out what was the problem and why did this fix it?

Reply:
You can try to test this by:
`--db '--postgres://bors:bors@localhost:5432/bors'` it will fail the same way :smile:

Reply:
Good find. With knowledge comes power :) Can you try if https://github.com/rust-lang/bors/pull/478 fixes the issue for you? If yes, we don't need to document anything.

Reply:
Thank you! It works smoothly  :smile:

Reply:
Awesome :)